crates/biome_cli/src/execute/process_file/lint.rs

```diff
@@ -22,10 +22,16 @@ pub(crate) fn lint_with_guard<'ctx>(
         move || {
             let mut input = workspace_file.input()?;
             let mut changed = false;
+            let (only, skip) =
+                if let TraversalMode::Lint { only, skip, .. } = ctx.execution.traversal_mode() {
+                    (only.clone(), skip.clone())
+                } else {
+                    (vec![], vec![])
```

Comment:
nit: ` (Vec::new(), Vec::new())` to be consistent with the other file

Reply:
Yes, I forget to remove this change.